app/assets/stylesheets/components/_published-dates.scss

```diff
@@ -2,6 +2,14 @@
   @include core-16;
 }
 
+.app-c-published-dates__toggle {
+  display: none;
+
+  .js-enabled & {
```

Comment:
Nit pick: not sure what our standards are but I've always seen the `&` selector used via the following format `&.js-enabled` - correct me if I'm wrong about is. 

Other than this, everything looks fine.

Reply:
My understanding is that they have a slightly different meaning? I may be wrong:

```
.element-one {
    &.element-two {}
}
```
This compiles to `.element-one .element-two`

Whereas
```
.element-one {
    .element-two & {}
}
```
compiles to `.element-two .element-one`

In this case, I think `js-enabled` is set on the `<body>`, so that's why the second variation is needed.